modules/optimum/tests/openvino/test_modeling_ov_auto.py

```diff
@@ -98,7 +98,62 @@ def test_from_ir(self):
         model = OVAutoModelForQuestionAnswering.from_pretrained(
             "dkurt/bert-large-uncased-whole-word-masking-squad-int8-0001"
         )
-        self.check_model(model, tok)
+        self.check_model(model, tok, "the garden")
+
+    @require_torch
+    def test_tinybert_from_pt(self):
+        model_name = "Intel/dynamic_tinybert"
+        tok = AutoTokenizer.from_pretrained(model_name)
+        model = OVAutoModelForQuestionAnswering.from_pretrained(model_name, from_pt=True)
+        self.check_model(model, tok, "if it makes me grow smaller, i can creep under the door")
+
+
+@require_torch
+class SameOutputTest(unittest.TestCase):
```

Comment:
Could we have more descriptive name? `MultiFrameworkSameOutputTest` or something like that?

Reply:
Yes, I'll change that, thanks!